ci_framework/roles/edpm_build_images/tasks/post.yaml

```diff
@@ -0,0 +1,38 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.  
+
+- name: Get latest commit when no PR is provided
```

Comment:
We don't need to get the latest commit, we have to find the delorean.repo.md5 sum hash : https://trunk.rdoproject.org/centos9-antelope/podified-ci-testing/delorean.repo.md5 

Also, i think that should be the input and not define here

Reply:
Done move this part to ci/playbooks/edpm_build_images/edpm_image_builder.yml